### DISCUSSION TITLE
Failed to connect to websocket

### DISCUSSION THREAD

**Discussion by acidiic** [rdeepak2002/reddit-place-script-2022#289] [Q&A]
Is this because of reddit or something I did

**Comment by rdeepak2002** [comment]
We are currently using the URL "wss://gql-realtime-2.reddit.com/query"

From using this tool: https://www.piesocket.com/websocket-tester I think that web socket url might be no longer working. 

<img width="742" alt="Screen Shot 2022-04-03 at 1 57 57 PM" src="https://user-images.githubusercontent.com/7110480/161441481-6f8f229a-0f51-4b10-bdfd-f7c72bad68fb.png">

**Reply by ap0th3cary** [reply]
Sure looks like you're right. Is there a workaround? Commenting out this section seems like a bad idea....
